src/Pages/ProductPage/Product.jsx

```diff
@@ -0,0 +1,97 @@
+import "./product.css";
+import "./product-Responsive.css";
+import { Nav } from "../../components/Nav/Nav";
+import { NavForMobile } from "../../components/NavForMobile/NavForMobile";
+import { Footer } from "../../components/Footer/Footer";
```

Comment:
export all components from index.js in components folder and import them as {Nav,Footer} from "../../components/index/"

Reply:
Thank you, I have Created Index file and exported it.